integration/emr/alluxio-emr.sh

```diff
@@ -170,13 +171,15 @@ main() {
   print_help() {
     local -r USAGE=$(cat <<USAGE_END
 
-Usage: alluxio-emr.sh <root-ufs-uri> [-b <backup_uri>]
-                             [ -d <alluxio-download-uri>]
-                             [-f <file_uri>]
-                             [-i <journal_backup_uri>]
-                             [-n <storage percentage>]
-                             [-p <delimited_properties>]
-                             [-s <property_delimiter>]
+Usage: alluxio-emr.sh [-u <root-ufs-uri>]
```

Comment:
alphabetical ordering of flags (but i may suggest using other characters in a bit)